package.json

```diff
@@ -91,47 +91,47 @@
     "express-jwt": "3.4.0",
     "font-awesome": "^4.6.3",
     "graphiql": "0.7.3",
-    "graphql": "0.6.0",
+    "graphql": "0.6.1",
     "graphql-custom-datetype": "^0.3.0",
     "isomorphic-fetch": "2.2.1",
-    "jsonwebtoken": "7.0.1",
+    "jsonwebtoken": "7.1.6",
     "jwt-decode": "^2.0.1",
-    "mailcomposer": "^3.9.0",
+    "mailcomposer": "^3.10.0",
     "mailgun-js": "^0.7.11",
     "ms": "^0.7.1",
     "node-env-flag": "0.1.0",
     "node-uuid": "1.4.7",
     "oy-vey": "^1.0.0-alpha",
-    "piping": "1.0.0-rc.2",
-    "react": "15.1.0",
-    "react-copy-to-clipboard": "^4.1.0",
-    "react-css-modules": "^3.7.6",
-    "react-dom": "15.1.0",
+    "piping": "1.0.0-rc.3",
+    "react": "15.2.1",
+    "react-copy-to-clipboard": "^4.2.1",
+    "react-css-modules": "^3.7.7",
+    "react-dom": "15.2.1",
     "react-fontawesome": "^1.1.0",
     "react-helmet": "^3.1.0",
     "react-hotkeys": "^0.8.0",
     "react-look": "file:./.mod_bin/react-look-1.0.1-parabol-patch1.tgz",
     "react-look-core": "file:./.mod_bin/react-look-core-1.0.1-parabol-patch1.tgz",
     "react-notification-system": "^0.2.7",
     "react-redux": "4.4.5",
-    "react-router": "2.5.1",
+    "react-router": "2.5.2",
     "react-router-redux": "4.0.5",
-    "react-textarea-autosize": "^4.0.3",
+    "react-textarea-autosize": "^4.0.4",
     "redux": "3.5.2",
     "redux-form": "^6.0.0-alpha.15",
     "redux-logger": "2.6.1",
     "redux-promise": "0.5.3",
-    "redux-socket-cluster": "0.5.1",
-    "redux-storage": "^4.0.1",
+    "redux-socket-cluster": "^0.8.0",
     "redux-storage-engine-localstorage": "^1.1.1",
+    "redux-storage-whitelist-fn": "^4.0.1",
```

Comment:
temporary package based on the PR i made that never got merged 😢 . No more save-on-every-freakin-everything.